include/SFML/System/Err.hpp

```diff
@@ -47,17 +47,17 @@ namespace sf
 /// \fn sf::err
 /// \ingroup system
 ///
-/// By default, sf::err() outputs to the same location as std::cerr,
+/// By default, sf::err() outputs to the same location as `std::cerr`,
```

Comment:
`sf::err` and other `sf::` types ought to get the same treatment but it’s up to you whether that happens now or in a subsequent PR

Reply:
Wasn't sure if we should do it, since Doxygen will automatically make a link out of SFML types. But I guess it might still be more readable (assuming it will still detect the links).

![image](https://github.com/user-attachments/assets/37edd6fe-6cf2-4125-910f-00547777bfa4)